server/internal/config/config.go

```diff
@@ -77,6 +81,9 @@ var (
 	UnlockerFilePath = "UNLOCKER_FILE_PATH"
 	UnlockerPassword = "UNLOCKER_PASSWORD"
 	NoteUriPrefix    = "NOTE_URI_PREFIX"
+	MarketStartTime         = "MARKET_START_TIME"
```

Comment:
let's keep the same prefix for consitency:

```suggestion
	MarketHourStartTime         = "MARKET_START_TIME"
```